g2p/tests/test_studio.py

```diff
@@ -97,7 +123,7 @@ async def test_langs(self):
                     for attempt in range(1, 4):
                         if attempt > 1:
                             LOGGER.info(f"Attempt #{attempt}")
-                            await page.wait_for_timeout(1000)
+                            await page.wait_for_timeout(5000)
```

Comment:
a 5 second timeout? were you running into problems with the 1 second one? this might make the tests take quite a while. Did you try other numbers in between?

Reply:
Ah, good catch, actually this is no longer needed now that I removed the deepcopy (I'm not sure that it actually waits 5 seconds, though)